src/codemirror/CodeMirror.vue

```diff
@@ -22,9 +22,9 @@ const props = withDefaults(defineProps<Props>(), {
 
 const emit = defineEmits<(e: 'change', value: string) => void>()
 
-const el = ref()
+const el = ref<HTMLDivElement>()
 const needAutoResize = inject('autoresize')
-const autoSave = inject('autosave')
+const autoSave = inject<boolean>('autosave')
```

Comment:
It's `Ref<boolean>` I guess.
```suggestion
const autoSave = inject<Ref<boolean>>('autosave')
```